spec/lcm/integration/user_filters_brick_spec.rb

```diff
@@ -206,22 +209,19 @@ def upload_user_filters_csv(user_filters)
     include_context 'a user action in sync_domain_client_workspaces mode'
     let(:user_filters) do
       [{
-           login: 'rubydev+admin@gooddata.com',
-           state: 'Washington',
-           client_id: 'testingclient'
+         login: 'rubydev+admin@gooddata.com',
```

Comment:
warning: Style/IndentHash: Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is. (rubocop)